src/z3c/dependencychecker/dependencychecker.py

```diff
@@ -343,6 +394,39 @@ def includes_from_zcml(path):
     return modules, test_modules
 
 
+def includes_from_plone_fti(path):
+    modules = []
+    test_modules = []
+    for path, dirs, files in os.walk(path):
+        for fti in [os.path.abspath(os.path.join(path, filename))
+                    for filename in files
+                    if fnmatch.fnmatch(filename, '*.xml')]:
+            contents = open(fti).read()
+            # check that we are on a Dexterity FTI and
+            # not any other random XML file
+            if contents.find('meta_type="Dexterity FTI"') == -1:
+                continue
+
+            found = [module for module in
+                     re.findall(FTI_BEHAVIOR_PATTERN, contents)
+                     if not module.startswith('.') and module.find('.') != -1]
```

Comment:
Why are we filtering matches with leading dots here? It shouldn't be possible to have relative dotted names here, it always must start with a dot in order to work properly, I think.